PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- Please verify that your code satisfies the [code/documentation quality guidelines](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines).
+- Please prefix the title of your pull request with `[MRG]` if the contribution is complete and should be subjected to a detailed review. Otherwise:
+- Incomplete contributions should be prefixed `[WIP]` to indicate a work in progress (and changed to `[MRG]` when it matures). WIPs may be useful to: indicate you are working on something to avoid duplicated work, request broad review of functionality or API, or seek collaborators. WIPs often benefit from the inclusion of a [task list](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) in the PR description.
```

Comment:
they aren't, and I've tried to change it as much as I could in the latest diff. It doesn't seem possible to create multi-line links, though (which is problematic because we have several long URLs)?